terraform/s3.tf

```diff
@@ -9,7 +9,6 @@ locals {
     aws_s3_bucket.pcc-master-data,
     aws_s3_bucket.pcc-deployments,
     aws_s3_bucket.pcc-s3-access-logs,
-    aws_s3_bucket.sftp_storage,
```

Comment:
I take it this was just fully redundant with sftp.tf ?

Reply:
From what I could tell the policies set to each bucket in this list were overriding the inline policy set from the sftp.tf file for this bucket which allows the sync job to run.